onnx/backend/test/runner/__init__.py

```diff
@@ -226,27 +226,21 @@ def assert_similar_outputs(
     def download_model(
         cls,
         model_test: TestCase,
-        model_dir: str,
```

Comment:
Is this a public interface? Should we make kwarg only?

Reply:
No, I think it's a test class. It should be safe to break BC.

Reply:
That would be ideal, but I’m not sure realistically what’s the state of usage. @xadupre does this look good to you?